src/components/Layout.tsx

```diff
@@ -51,11 +51,18 @@ export function Layout({ children, sections = [], disableEditGithub = false }) {
                   </Note>
                 ) : (
                   <Note>
-                    {currentSdkLabel} is in development for v1. Interested in
-                    contributing or chatting with us?
+                    The {currentSdkLabel} SDK currently only supports legacy
+                    versions of Nitric prior to v1. This version is maintained
+                    for compatibility with existing projects and not recommended
+                    for new projects. New projects should be started using a{' '}
+                    <Link className="ml-1" href="/#libraries">
```

Comment:
This link goes to a component that still has the legacy sdks on it. Could just remove the link and suggest using the command only.